docs/index.html

```diff
@@ -137,6 +175,7 @@ <h2>#TwitterThrowback</h2>
     let hash = window.location.hash.split('#')[1];
     if (hash != '' && hash != undefined) {
         usernameInput.value = hash;
+        handleFormSubmit() //todo
```

Comment:
This function is supposed to take an event parameter. Please remove the function call for now. You've done good work here. We can then update the function and add this in a separate PR.

Reply:
damn, wanted to remove that, I forgot, lol.

Reply:
done